prime-dotnet-webapi/Migrations/20230208234852_AllNursesHavePharmanetId.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class AllNursesHavePharmanetId : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropCheckConstraint(
+                name: "CHK_Agreement_EitherPartyOrEnrollee",
+                table: "Agreement");
+
```

Comment:
I think you also have to add `prime-dotnet-webapi/Configuration/Database/Unseeded Tables/AgreementConfiguration.cs` to `.gitattributes`